_guides/stylesheet/config.css

```diff
@@ -157,3 +157,8 @@ table.configuration-reference.tableblock .description-decoration i {
 }
 
 .configuration-legend span.icon { color: #0D1C2C; }
+
+blockquote {
+  color: #808080;
+  font-style: italic;
+}
```

Comment:
This is the CSS for the configuration reference table. I will close this PR and have a look now on how to best fix this.

Thanks for catching it!

Reply:
Ok. Closing it